tools/hive-bigquery/init_script.py

```diff
@@ -0,0 +1,147 @@
+             "gs://BUCKET_NAME or BUCKET_NAME. Hive data will be copied to this"
+             "bucket and then loaded into BigQuery")
+    optional.add_argument(
+        '--incremental-col', required=False,
+        help="Provide the incremental column name if present in the Hive table")
+    optional.add_argument(
+        '--use-clustering', required=False,
+        default="False", choices=["False", "True"],
+        help="Boolean to indicate whether to use clustering in BigQuery if "
+             "supported")
+    required.add_argument(
+        '--tracking-database-host', required=False, default='localhost',
+        help="Cloud SQL Tracking database host address, defaults to localhost")
+    optional.add_argument(
+        '--tracking-database-port', required=False, default=3306,
+        help="Port which you used for running cloud sql proxy, defaults to 3306")
+    optional.add_argument(
+        '--tracking-database-user', required=False, default='root',
+        help="Cloud SQL Tracking database user name, defaults to root")
+    required.add_argument(
+        '--tracking-database-db-name', required=True,
+        help="Cloud SQL Tracking db name.Ensure you provide the same db name "
+             "if you have migrated previously")
+
+    return parser.parse_args()
+
+
+def initialize_variables():
+    """Initialize variables"""
+
+    args = arg_pars()
+
+    # Gets password from the user to coonect to Cloud SQL tracking database
+    tracking_database_password = getpass.getpass(
+        'Enter Cloud SQL tracking database password:')
+
+    time_format = datetime.datetime.now().strftime("%Y_%m_%d_%H_%M_%S")
+    hive_bq_comparison_csv = args.hive_table + "_metrics_hive_bq_" + \
+                             time_format + '.csv'
+    hive_bq_comparison_table = args.hive_table + "_metrics_hive_bq_" + \
+                               time_format
+    log_file_name = args.hive_table + "_metrics_hive_bq_" + time_format + '.log'
+
+    logger = logging.getLogger('Hive2BigQuery')
+    logger.setLevel(logging.DEBUG)
+    # Sets log formatter with time,log level,filename,line no and function
+    # name which produced that log statement
+    formatter = logging.Formatter(
+        "[%(asctime)s - %(levelname)s - %(filename)25s:%(lineno)s - %("
+        "funcName)20s() ] %(message)s")
+    log_handler = logging.FileHandler(filename=log_file_name)
+    log_handler.setLevel(logging.DEBUG)
+    log_handler.setFormatter(formatter)
+    # Adds the above defined handler
+    logger.addHandler(log_handler)
+
+    # BigQuery table name defaults to Hive table name if not provided
+    if args.bq_table == '':
+        args.bq_table = args.hive_table
+
+    if args.gcs_bucket_name.startswith('gs://'):
+        args.gcs_bucket_name = args.gcs_bucket_name.split('gs://')[1]
+    if args.gcs_bucket_name[-1] == '/':
+        args.gcs_bucket_name = args.gcs_bucket_name[:-1]
+
+    configuration_properties = {
```

Comment:
Why don't we just have the user pass this exact json file? will make things simpler.